syntax/crisp_lexer.mll

```diff
@@ -11,6 +11,40 @@
 open Lexing
 open Crisp_parser
 
+exception Error
+
+let backtrack str lexbuf =
+  lexbuf.lex_curr_pos <- lexbuf.lex_curr_pos - (String.length str)
+
+let count_nl str =
+  if str <> (String.make (String.length str) '\n') then
```

Comment:
Is it equivalent to use `"\n"` instead of `(String.make (String.length str) '\n')`?